[BUILD] When run `mvn install` it fails with license check

```
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.16.1:check (default) on project auron-common_2.12: Too many files with unapproved license: 1 See RAT report in: /Code/auron/common/target/rat.txt -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :auron-common_2.12
```

--- a/.rat-excludes
+++ b/.rat-excludes
@@ -32,3 +32,4 @@
 NOTICE*
 docs/**
 build/apache-maven-*/**
+src/main/resources/auron-build-info.properties

